tox.ini

```diff
@@ -226,8 +232,8 @@ deps =
 
 changedir = docs
 
-commands_pre =
-  python -m pip install {toxinidir}/opentelemetry-python-contrib/opentelemetry-instrumentation
+commands-pre =
+  python -m pip install {toxinidir}/opentelemetry-instrumentation
```

Comment:
```suggestion
  pip install {toxinidir}/opentelemetry-instrumentation
```

Reply:
We should have `pip` installed: 

https://github.com/open-telemetry/opentelemetry-python/blob/dfd298064b20ed83622201abde1a738e658509a2/tox.ini#L118